modules/distribution/product/identity_config_change.xml

```diff
@@ -12,6 +12,11 @@
       <after>//s:Server/s:OAuth/s:OAuthCallbackHandlers</after>
       <value><![CDATA[<OAuthScopeValidator class="org.wso2.carbon.identity.oauth2.validators.JDBCScopeValidator"/>]]></value>
    </add>
+
+    <remove>
+        <name>//s:Server/s:OAuth/s:ScopeValidators</name>
+    </remove>
```

Comment:
formatting issue with indentation

Reply:
The added part doesn't have any formatting issue. Remains L2-L4, L6-L11, L13-L17, L23-L30 lines have the formatting issue